src/config.ts

```diff
@@ -161,6 +165,7 @@ export default {
     ENABLE_COMMERCE_STITCHING === undefined
       ? true
       : ENABLE_COMMERCE_STITCHING === "true",
+  ENABLE_CAUSALITY_STITCHING: ENABLE_CAUSALITY_STITCHING === "true",
```

Comment:
Don't think this is necessary. These flags were useful in the past when stitching was experimental, and if we needed to we could flip it off and fall back to regular loader patterns. 

Since there aren't loaders for Causality that map 1:1 to what is being stitched, we don't need a flag since setting it to `false` would break consumers.